### ISSUE TITLE
Added banner, colored output, and improved argument usage 

### ISSUE THREAD

**Issue by cTFk1ller** [AyalX/Instagram-Lookup#2]
### Hey there! 

- I just made some updates to the code in the README.md file
- I added a banner to the top of the script, colored the output for better readability
- I improved the argument usage to make it more convenient for users. 

#### These changes should make the code more user-friendly and visually appealing.  [Check forked](https://github.com/cTFk1ller/Instagram-Lookup)

**Let me know if you have any questions or feedback!**

**Comment by AyalX** [comment]
Awesome! Shouldn't we just merge them?

**Comment by cTFk1ller** [comment]
> Awesome! Shouldn't we just merge them?

Yes, accept the changes in the pull requests section of the "Instagram-lookup" repo.

**Comment by AyalX** [comment]
Of course, but it seems you haven't made a pull request yet

**Comment by cTFk1ller** [comment]
Yea, Please check again. I made the pull request.

**Comment by AyalX** [comment]
I have merged the pull request now, though it seems there's parsing issues:
![image](https://github.com/AyalX/Instagram-Lookup/assets/25435680/cadfabaa-d61c-4e2c-99b5-35c74bc9f492)

**Comment by cTFk1ller** [comment]
Okay, it looks like the issue is related to `colouring the output` because you are not using **a Linux or Mac terminal** but a **Windows cmd**.

![FotoJet](https://github.com/AyalX/Instagram-Lookup/assets/110791478/e91c2c56-2a06-40e6-a683-c22e2652d3b1)

**`I will solve this issue, no worries.`**
